dln/operator.py

```diff
@@ -167,8 +168,9 @@ def __init__(self, model_name: str = "text-davinci-003", **generation_options):
         super().__init__(model_name, **generation_options)
         engine_for_encoder = self.engine.replace("gpt-35", "gpt-3.5")
         self.encoder = instantiate_tokenizer(engine_for_encoder)
-        openai.api_version = os.environ.get('OPENAI_API_VERSION')
         self._has_logprobs = self.engine in self.LOGPROBS_MODELS
+        self.client = openai.OpenAI()
+        self.aclient = openai.AsyncOpenAI()
```

Comment:
We probably needs to deal with Azure here.

e.g. I used this
`client = openai.AzureOpenAI() if openai.api_type == "azure" else openai.OpenAI()`